TikTokTrainer/TikTokTrainerApp.swift

```diff
@@ -10,6 +10,24 @@ import Promises
 
 @main
 struct TikTokTrainerApp: App {
+    
+//  for testing
+//    init() {
+//        var preRecordedVid = PoseNetProcessor.run(url: Bundle.main.url(forResource: "workout", withExtension: "mp4")!)
+//        var recordedVid = PoseNetProcessor.run(url: Bundle.main.url(forResource: "f", withExtension: "mov")!)
+//        var function = ScoringFunction()
+//        preRecordedVid.then { vid in
+//            function.preRecordedVid = vid
+//            recordedVid.then { vid in
+//                function.recordedVid = vid
+//                var score = function.computeScore()
+//                score.then { score in
+//                    print("score: " + score.description)
+//                }
+//            }
+//        }
+//    }
+
```

Comment:
We can save this externally (I'll make an issue with the code that we can copy/paste) so it doesn't need to be in develop